src/feedClass.ts

```diff
@@ -0,0 +1,281 @@
+import * as _ from 'lodash'
+import Gun from 'gun'
+import InterfaceFeed from './interfaces/interfaceFeed'
+import {Feed, FeedInfo, FeedItem, Feeds} from './types'
+import gunDB from './lib/gundb'
+import logger from './lib/logger'
+
+
+export default class feedClass implements InterfaceFeed {
+    gun = gunDB;
+    gunUser = this.gun.user();
+    feeds = null;
+
+    feedItemDataFields = [
```

Comment:
Fields lists should be defined as private readonly and, probably static as those are just constants